src/process.rs

```diff
@@ -115,3 +120,34 @@ pub fn init(ctx: &Ctx<'_>) -> Result<()> {
 
     Ok(())
 }
+
+pub struct ProcessModule;
+
+impl ModuleDef for ProcessModule {
+    fn declare(declare: &mut Declarations) -> Result<()> {
+        declare.declare("env")?;
+        declare.declare("cwd")?;
+        declare.declare("argv0")?;
+        declare.declare("id")?;
+        declare.declare("argv")?;
+        declare.declare("platform")?;
+        declare.declare("arch")?;
+        declare.declare("hrtime")?;
+        declare.declare("release")?;
+        declare.declare("version")?;
+        declare.declare("versions")?;
+        declare.declare("exit")?;
+
+        declare.declare("default")?;
+        Ok(())
+    }
+
+    fn evaluate<'js>(ctx: &Ctx<'js>, exports: &mut Exports<'js>) -> Result<()> {
+        let globals = ctx.globals();
+        let process: Object = globals.get("process")?;
+
+        exports.export("default", process)?;
```

Comment:
This won't allow you to access the named exports from process tho:
`import {env} from "process"`

Reply:
I addressed it, spot on.